CONTRIBUTING.rst

```diff
@@ -228,3 +227,20 @@ To run tests in parallel on eight processors::
 To run tests verbosely, displaying all print statements to terminal::
 
     py.test  -vvs
+
+Things to Keep in Mind When Contributing
+----------------------------------------
+
+#. An update to the ``CHANGELOG`` is required if any important changes are made that needs to be communicated such as:
```

Comment:
This shouldn't be in a different section, instead include this in https://github.com/nexB/scancode-toolkit/blob/70564a4a513d46eaa1fc0deff1cf1b61200df940/CONTRIBUTING.rst#development between points 5 and 6.

Reply:
Added. Can you look into the changes?😀